examples/BicepHelpers/actionGroup.bicep

```diff
@@ -1,9 +1,5 @@
-/*
```

Comment:
Two things... this file was originally supposed to be an example of a failing test for testing `Confirm-BicepFile` (it's demonstrated in the `Confirm-BicepFile` comments). It was fixed when Megalinter was introduced (hard coded `location` and unused variable $location - just change the line `location: location` to `location: 'eastus'` and it creates both problems). I think the failure case might still be relevant for testing, but it fails Megalinter. But fixing the file for Megalinter makes the file useless for negative testing `Confirm-BicepFile`, which is the current state. So, a bit of a conundrum.

The second thing is, why is this file in the BicepHelpers folder?

Reply:
> I noticed you removed the spin-up/tear-down tests. I can see why they'd be problematic. Can you elaborate on your decision to do that?

To answer two questions at once, I removed the spin-up/tear-down tests from each `*.Tests.ps1` file because I felt they were repetitive to have in every single `*.Tests.ps1` file when the only things that were changing were the parameters being passed into `Deploy-` and `Remove-BicepFeature`.

So I created a BicepHelpers folder to show examples for the "bicep" cmdlets (`Deploy-`/`Remove-BicepFeature` and `Confirm-BicepFile`). There are additional steps in the document talking about how to edit the `actionGroup.bicep` file to force a failure and use it with `Confirm-BicepFile` to run checks. Essentially, any bicep file could be used with these "bicep helpers", but I just arbitrarily used the `actionGroup.bicep` file. This should hopefully solve the conundrum of checking in code that would fail megalinter.

Lastly, will remove the `actionGroup.bicep` file since I think I accidentally left it in and instead edit `BicepHelpers.Tests.ps1` to reference the already existing bicep file in the `ActionGroup` folder.